### PR TITLE
[xtreme-s] Update Minds14 results

### PR DESCRIPTION
**Pull request by anton-l**
# What does this PR do?

Updated results for the XTREME-S experiments with the new Minds14 splits

### CODE REVIEW

**Comment by patrickvonplaten** [comment]
Let's maybe put the new eval code directly into this here as well?

File: examples/research_projects/xtreme-s/run_xtreme_s.py [hunk 1]
```diff
@@ -816,7 +831,20 @@ def compute_classification_metric(pred):
     results = {}
     if training_args.do_predict:
         logger.info(f"*** Evaluating on the `{data_args.predict_split_name}` set ***")
-        metrics = trainer.evaluate(vectorized_datasets["predict"])
+        if data_args.per_lang_metrics:
```
Comment (patrickvonplaten) [inline examples/research_projects/xtreme-s/run_xtreme_s.py:834]: nice!

File: examples/research_projects/xtreme-s/README.md [hunk 2]
```diff
@@ -67,7 +67,7 @@ The corresponding training commands for each dataset are given in the sections b
 | Speech Recognition    | VoxPopuli | -                     | -                                                                  | -             | -      |
 | Speech Recognition    | FLEURS    | -                     | -                                                                  | -             | -      |
 | Speech Translation    | CoVoST-2  | -                     | -                                                                  | -             | -      |
-| Speech Classification | Minds-14  | 94.74 F1 / 94.70 Acc. | [here](https://huggingface.co/anton-l/xtreme_s_xlsr_300m_minds14/) | 04:46:40      | 2xA100 |
+| Speech Classification | Minds-14  | 90.56 F1 / 90.55 Acc. | [here](https://huggingface.co/anton-l/xtreme_s_xlsr_300m_minds14/) | 2:54:21       | 2xA100 |
```
[the remark this answers is not in the document]
Comment (anton-l) [reply examples/research_projects/xtreme-s/README.md]: Updated :+1:

**Comment by anton-l** [comment]
@patrickvonplaten The per-language results are now reflected in the model card too: https://huggingface.co/anton-l/xtreme_s_xlsr_300m_minds14 (will update the MLS results after running one more full experiment).
The names look a bit janky due to how the metrics are parsed by the trainer, but overall the PR is ready to merge :slightly_smiling_face:

**Review by patrickvonplaten — APPROVED** [review]

**Pull request merged**